packages/imperative/__tests__/config/cmd/report-env/Report-env.handler.unit.test.ts

```diff
@@ -0,0 +1,106 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import ReportEnvHandler from "../../../../src/config/cmd/report-env/Report-env.handler";
+import { EnvQuery, IGetItemVal } from "../../../../src/config/cmd/report-env/EnvQuery";
+import { ItemId } from "../../../../src/config/cmd/report-env/EnvItems";
+
+describe("Handler for config report-env", () => {
+
+    let rptEnvHandler: any; // use "any" so we can call private functions
+
+    beforeAll(() => {
+        // instantiate our handler
+        rptEnvHandler = new ReportEnvHandler();
+    });
+
+    describe("process method", () => {
+
+        // fake command parms passed to process()
+        let stdoutMsg = "";
+        let stderrMsg = "";
+        let apiMessage = "";
+        let jsonObj;
+        const cmdParms = {
+            arguments: {
+                $0: "fake",
+                _: ["fake"]
+            },
+            response: {
+                data: {
+                    setMessage: jest.fn((setMsgArgs) => {
+                        apiMessage = setMsgArgs;
+                    }),
+                    setObj: jest.fn((setObjArgs) => {
+                        jsonObj = setObjArgs;
+                    }),
+                    setExitCode: jest.fn((exitCode) => {
+                        return exitCode;
+                    })
+                },
+                console: {
+                    log: jest.fn((logArgs) => {
+                        stdoutMsg += "\n" + logArgs;
+                    }),
+                    error: jest.fn((logArgs) => {
+                        stderrMsg += "\n" + logArgs;
+                    })
+                },
+                progress: {}
+            },
+            profiles: {}
+        } as any;
+
+        it("should report the output of getEnvItemVal", async () => {
+            stdoutMsg == "";
+            stderrMsg == "";
```

Comment:
What an awful typo! I do want to keep the assignments. With just the one test, the assignments are not necessary. Since the mock response.console.log() appends every message, they would accumulate across tests if we added similar tests. It is good to initialize the output messages to nothing before a test that wants to check that output. Hopefully this establishes a pattern that one would follow if another test is added in the future. I will definitely fix the double equals.